nucliadb/src/nucliadb/search/api/v1/resource/search.py

```diff
@@ -76,7 +76,6 @@ async def resource_search(
     highlight: bool = fastapi_query(SearchParamDefaults.highlight),
     x_ndb_client: NucliaDBClientType = Header(NucliaDBClientType.API),
     debug: bool = fastapi_query(SearchParamDefaults.debug),
-    shards: list[str] = fastapi_query(SearchParamDefaults.shards),
```

Comment:
Same as above ^

Reply:
I don't think we even need to communicate removal from Find/Ask Request. If they send it, they will be ignored and behave exactly as they do now. That is, nothing changes at all for the user.

Reply:
the "problem" is that those same model defs are also being used on the python sdks, so if someone blindly upgrades will start failing. maybe I am being too paranoid now